Yank/tests/test_yaml.py

```diff
@@ -1439,7 +1438,6 @@ def test_setup_explicit_solvation_system():
             assert os.path.getsize(inpcrd_path) > 0
             assert found_resnames == expected_resnames[phase]
 
-
```

Comment:
This is very minor, but PEP8 suggests to separate top level functions with 2 blank lines: https://www.python.org/dev/peps/pep-0008/#blank-lines